src/ir/layout.rs

```diff
@@ -111,10 +112,10 @@ impl CanTriviallyDeriveDebug for Opaque {
     }
 }
 
-impl<'a> CanDeriveDefault<'a> for Opaque {
+impl CanTriviallyDeriveDefault for Opaque {
     type Extra = ();
 
-    fn can_derive_default(&self, _: &BindgenContext, _: ()) -> bool {
+    fn can_trivially_derive_default(&self, _: &BindgenContext, _: ()) -> bool {
```

Comment:
it seems like this is the only impl, and since it is not using the `Extra` associated type, nor the `ctx` parameter, we should remove them.